docs/HOWTO/20_Puppet/Hiera_eyaml.rst

```diff
@@ -0,0 +1,189 @@
+.. _howto-set-up-and-utilize-hiera-eyaml:
+
+HOWTO Set up and Utilize Hiera-eyaml
+====================================
+
+This section steps through configuring the :term:`hiera-eyaml`  backend
+for a puppet environment.  This backend allows you to encrypt data in
+the :term:`Hiera` files.
+
+An example of how to use the :command:`eyaml` command provided by the :term:`hiera-eyaml`
+gem to encrypt data is included at the end.
+
+More details on configuring and using eyaml can be found in the puppet
+documents for `Configuring hiera-eyaml`_ and in the `hiera-eyaml documentation`_.
+
+Install the gem.
+----------------
+
+The hiera-eyaml backend has been included in Puppet Server since version 5.2.0.
+If the Puppet Server does not already have the hiera-eyaml gem installed run:
+
+.. code-block::  bash
+
+   $ puppetserver gem install hiera-eyaml
+
```

Comment:
It seems like anyone running SIMP 6.6.0 would already have a Puppet Server with `hiera-eyaml`.

SIMP has shipped with puppetserver >= 5.2 since 6.3.0-0 (Jan 2019) and PE started with 2017.3.
That means the `puppetserver gem install hiera-eyaml` only applies to sites with an EOL Puppet Server and no plans to upgrade to SIMP 6.6.0 (which requires Puppet server 5.3+), which seems quite edge case-y.  

I'm tempted to remove it or move it out of the main workflow and into a later section for edge cases or troubleshooting.